qfieldsync/core/cloud_api.py

```diff
@@ -222,6 +318,27 @@ def set_auth(self, url, **kwargs: str) -> None:
             auth_manager.storeAuthenticationConfig(cfg)
             self.preferences.set_value("qfieldCloudAuthcfg", cfg.id())
 
+    def auth_provider_id(self) -> str:
+        """
+        Returns the provider ID, required for sso auth to QFieldCloud.
+        Should be stored in the QgsAuthMethodConfig with the "qfieldcloud_sso_id" key.
+        """
+        auth_method = self.auth()
+        if not auth_method:
+            return ""
```

Comment:
```suggestion

        if not auth_method:
            return ""

```

A bit of air. Make sure the early returns are prominently surrounded by empty lines.